reference/features/telemetry.md

```diff
@@ -0,0 +1,50 @@
+# Telemetry
+
+MeiliSearch collects anonymized data from users in order to improve our product. This can be [deactivated at any time](#how-to-disable-data-collection).
+
+Note that while MeiliSearch is running in [`development` mode](/reference/features/configuration.md#environment), no data is collected.
+
+## What kind of data do you collect?
+
+We use two data-collection tools: [Amplitude](https://amplitude.com/amplitude-analytics) and [Sentry](https://sentry.io/welcome/). Both can be disabled by [following the below procedure](#how-to-disable-data-collection).
+
+**Amplitude** collects information about how and where MeiliSearch is being used, including the amount of time a MeiliSearch instance has been running, the version it's using, the device that's running it, and the number of documents per index. It also tracks the number of unique users, which is our equivalent of counting downloads or installs.
+
+Note that this data is aggregated, meaning that we never view individual statistics.
```

Comment:
I'm not 100% sure this is true, can anyone confirm?

Reply:
Also `statistics` is probably the wrong word 😢

Reply:
> Also `statistics` is probably the wrong word 😢

Does the term `metrics` can be a better fit?

Reply:
> I'm not 100% sure this is true, can anyone confirm?

In fact, we have the possibility to have access to a "user" (instance) page, this is what Amplitude calls ["User Look-up"](https://help.amplitude.com/hc/en-us/articles/229313067-User-Look-Up#User-Profile).

In practice, the data we visualize will always be aggregated data. It is possible to see custom fields values in the page of a "user", in our case, what I call custom fields are the number of days of use, or the version of MeiliSearch. These custom fields are then used to segment the "user" in a specific cohort.

This page does not allow us to identify a user with the data that we store.

TLDR; We can view individual data per "user" (instance) despite the fact that we don't store anything that allows us to identify a person or a group of people. That specific view doesn't bring anything of value in its current state for the core product.

Reply:
Understood, thank you very much @gmourier for the clarification! 😄